code/API_definitions/location-verification.yaml

```diff
@@ -32,6 +32,12 @@ info:
 
     * **Area**: It specifies the geographical surface where a device may be physically located.
 
+     * **Max Age**: Maximum age of the location information which is accepted for the location verification (in seconds).
+        * Absence of maxAge means "any age" is acceptable for the client. In other words, this is like maxAge=infinite
+        * maxAge=0 means a fresh calculation is requested by the client.
+          If the system does not support it, or fresh location cannot be checked at that time for any reason, the API response will be "unknown".
```

Comment:
I think this behavior can be helpful to add:
"In this case. the system may still return lastLocationTime, if available."

Reply:
We already indicate that above, with "the verification result will be `UNKNOWN` and the `lastLocationTime` attribute may indicate the last available time for the device location", but it is not bad to reinforce the guideline here